src/ape/api/accounts.py

```diff
@@ -137,6 +170,19 @@ def transfer(
         return self.call(txn, send_everything=value is None)
 
     def deploy(self, contract: ContractContainer, *args, **kwargs) -> ContractInstance:
+        """
+        Creates a smart contract on the blockchain.
+
+        Method Limiations:
```

Comment:
```suggestion
        Method Limitations:
```

^ Not actually sure how this looks on the docs site